code/controllers/subsystems/chat.dm

```diff
@@ -7,7 +7,6 @@ SUBSYSTEM_DEF(chat)
 
 	var/list/payload = list()
 
```

Comment:
/datum/controller/subsystem/chat/Initialize()
	. = ..()
	
Should put this somewhere so it can initialize at least